src/ugeneui/src/Main.cpp

```diff
@@ -520,6 +538,29 @@ int main(int argc, char **argv) {
     if (!translator.isEmpty()) {
         QCoreApplication::installTranslator(&translator);
         GObjectTypes::initTypeTranslations();
+#ifdef Q_OS_MAC
+    } else {
+        CFURLRef appUrlRef = CFBundleCopyBundleURL(CFBundleGetMainBundle());
```

Comment:
You fixed this part for ugenecl but left it unchanged for ugeneui

Reply:
done